eng/Versions.props

```diff
@@ -4,14 +4,18 @@
   <PropertyGroup>
     <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
   </PropertyGroup>
-  <!-- Opt out of certain Arcade features -->
-  <PropertyGroup>
+  <PropertyGroup Label="Opt out of certain Arcade features">
```

Comment:
Some of these groups had comments that are being used identify the contents of the group. So, I just changed the comments into actual group labels.